Makefile

```diff
@@ -1,70 +1,37 @@
-	go run main.go reset --confirm
-	@cd client && go clean -testcache && go test ./... -v -run $(TEST)
-
-coverage:
-	@cd client && go clean -testcache && go test ./... -coverprofile=coverage.out && go tool cover -html=coverage.out && rm coverage.out
-
-init-test-env:
-	docker compose up -d
-	@echo "> Waiting 45 seconds for ElasticSearch to fully boot..."
-	@sleep 45
-	curl -X PUT -H "Content-Type: application/json" http://localhost:9400/_cluster/settings -d '{ "transient": { "cluster.routing.allocation.disk.threshold_enabled": false } }'
-
-create-es-indices:
-	curl -X PUT http://localhost:9400/biblio_publications_20060102150405 -H 'Content-Type: application/json' -d @etc/es6/publication.json
-	curl -X POST http://localhost:9400/_aliases -H 'Content-Type: application/json' -d '{"actions":[{"add":{"index":"biblio_publications_20060102150405","alias":"biblio_publications"}}]}'
-	curl -X PUT http://localhost:9400/biblio_datasets_20060102150405 -H 'Content-Type: application/json' -d @etc/es6/dataset.json
-	curl -X POST http://localhost:9400/_aliases -H 'Content-Type: application/json' -d '{"actions":[{"add":{"index":"biblio_datasets_20060102150405","alias":"biblio_datasets"}}]}'
-	curl -X PUT http://localhost:9400/biblio_person -H 'Content-Type: application/json' -d @etc/es6/person.json
-	curl -X PUT http://localhost:9400/biblio_project -H 'Content-Type: application/json' -d @etc/es6/project.json
-	curl -X PUT http://localhost:9400/biblio_organization -H 'Content-Type: application/json' -d @etc/es6/organization.json
-	curl -X PUT -H "Content-Type: application/json" http://localhost:9400/_all/_settings -d '{"index.blocks.read_only_allow_delete": null}'
-	curl -X POST http://localhost:9400/biblio_person/person/_bulk -H "Content-Type: application/x-ndjson" --data-binary @etc/fixtures/person.es6.jsonl
-	curl -X POST http://localhost:9400/biblio_organization/organization/_bulk -H "Content-Type: application/x-ndjson" --data-binary @etc/fixtures/organization.es6.jsonl
-
-create-mongo-db:
-	mongoimport --uri "mongodb://localhost:27020/authority?directConnection=true&serverSelectionTimeoutMS=2000" --collection person ./etc/fixtures/person.jsonl
-
-create-db-tables:
-	go install github.com/jackc/tern@latest
-	tern migrate --migrations etc/snapstore/migrations
-
-api-server:
-	go run main.go api start
-
-
-# Declare all targets as "PHONY", see https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html.
-MAKEFLAGS += --always-make
-.PHONY: init-test-env create-es-indices create-db-tables api-server setup-test-env tear-test-env run-test-grpc-client
\ No newline at end of file
+vTempl=github.com/a-h/templ/cmd/templ@v0.2.747
+vWgo=github.com/bokwoon95/wgo@latest
+
+live/templ:
+	@go run $(vTempl) generate \
+	--path views \
+	--watch \
+	--proxy="http://localhost:3011" \
+	--proxyport=3001 \
+	--proxybind="localhost" \
+	--open-browser=false
+
+live/esbuild:
+	@node esbuild.mjs --watch
+
+live/server:
+	@BIBLIO_BACKOFFICE_PORT=3011 go run $(vWgo) run \
```

Comment:
I chose to run the server (in live mode only) on port 3011 so the templ proxy can run on 3001 which aligns with dev mode.